src/test/java/org/isf/patient/test/Tests.java

```diff
@@ -216,6 +223,17 @@ public void testIoGetPatientAll() throws Exception {
 		assertThat(patient.getName()).isEqualTo(foundPatient.getName());
 	}
 
+	@Test
+	public void testIoGetPatientsByParams() throws Exception {
+		_setupTestPatient(false);
+		Map<String, Object> params = new HashMap<String, Object>();
+		params.put("firstName", "TestFirstN");
+		params.put("birthDate", new GregorianCalendar(1984, Calendar.AUGUST, 14).getTime());
+		params.put("address", "TestAddress");
+		ArrayList<Patient> patients = patientIoOperation.getPatients(params);
+		assertThat(patients.size() > 0).isTrue();
```

Comment:
A more idiomatic version would be:  **assertThat(patients.size()).isPositive();**

Reply:
Yes! I've modified it.